terraform/dev/terragrunt.hcl

```diff
@@ -43,9 +43,12 @@ generate "dev_tfvars" {
   fam_console_idp_name_bceid = "TEST-BCEIDBUSINESS"
   forest_client_api_base_url = "https://nr-forest-client-api-test.api.gov.bc.ca"
   use_override_proxy_endpoints = true
-  dev_override_bcsc_userinfo_proxy_endpoint = "https://c727z9v3cc.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/dev"
-  test_override_bcsc_userinfo_proxy_endpoint = "https://c727z9v3cc.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/test"
-  prod_override_bcsc_userinfo_proxy_endpoint = "https://c727z9v3cc.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/prod"
+  dev_override_bcsc_userinfo_proxy_endpoint = "https://6mud7781pe.execute-api.ca-central-1.amazonaws.com/v1/bcsc/userinfo/dev"
```

Comment:
use the AWS PROD (PROD API Gateway) token url and user info url to config dev/test/prod BCSC in AWS DEV, so we can use AWS DEV cognito to test the PROD-BCSC login

Reply:
I think the use_override_proxy_endpoints is not related to the flyway, we only use it here when set up token and user info urls